codeformat.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+DIR=$(cd "$(dirname "$0")"; pwd -P)
+
+black "$DIR"
+flake8 "$DIR"
```

Comment:
In middleware people use pre-commit hooks which work reasonably OK and do not require separate script which you can forget to run. We use `pre-commit` package: https://pre-commit.com/, and you can check an example of configuration here: https://github.com/lsst/daf_butler/blob/main/.pre-commit-config.yaml.

Reply:
:+1: